test-cases/scale/longevity-elasticity-many-small-tables.yaml

```diff
@@ -0,0 +1,36 @@
+# This is a test case for having 90% disk utilization with a lot of small tables.
+# The data is split equally among 500 tables.
+# The dataset size is aligned with 'i4i.xlarge'.
+# It uses a c-s user-profile template for all 500 tables.
+# It runs 4 batches of 125 tables each.
+# On each batch cycle, 125 tables are created, then a load is generated for all of these tables.
+# When all the 125 stress writes/reads are done, it continue with the next batch until stress to all 500 tables is completed (after 4 cycles).
+# Each one of the 500 tables has both write and read load.
+
+test_duration: 480
```

Comment:
I think this might be too tight, base on the run you shared (and timeouted)

I would recommend leaving enough slack for possible regressions on how long each cycle would take

Reply:
passed ok with 600.